src/main/java/org/commonwl/view/cwl/CWLValidationException.java

```diff
@@ -19,7 +19,7 @@
 
 package org.commonwl.view.cwl;
 
-import javax.validation.ValidationException;
+import jakarta.validation.ValidationException;
```

Comment:
Note: Iacopo might have to test a deploy of this version, to make sure the server is using jakarta and not javax too.